contracts/tgrade-valset/src/contract.rs

```diff
@@ -532,6 +538,19 @@ fn end_block(deps: DepsMut, env: Env) -> Result<Response, ContractError> {
     // calculate and store new validator set
     let (validators, auto_unjail) = calculate_validators(deps.as_ref(), &env)?;
 
+    // update operators list with info about whether or not they're active validators
+    let ops: Vec<_> = operators()
```

Comment:
This is the beginning of the problem.
the validators set is bounded: (eg 100 or 300) based on config settings.
Operators list is unbounded (anyone can register, this stays forever)... it can be 10.000 easily if people register.

Do not range this